### PR TITLE
Fault Injection Service Fabric Hackathon (Intergen)

### PR DESCRIPTION
**Pull request by steven2000nz**

### CODE REVIEW

**Comment by steven2000nz** [comment]
Hi Oguz:
How are you? Thanks for reviewing this lab. 
I've tried this lab myself, it seems ok. 
Can we please check a few things at your end?
1. node.exe is in folder **C** (Task 2, Step 6)
2. .env.example is configure to use MongoDB (Task 3, Step 7)
3. Folder and File structure looks like the following:
image
image

Cheers
Steven

File: docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+cd hackathon-starter
+
+# Install NPM dependencies
+npm install
+```
+
+**Step 4.** We need to make sure MongoDB is running before we try and start the application. Open a second command prompt and run the following command
+
+`mongod`
+
```
Comment (gavinbarron) [inline docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md]: Done

File: docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+                |-- Hackathon application files (the ones you got from Github)
+                |-- node.exe
+            |-- config
+                |-- Settings.xml
+            |-- ServiceManifest.xml
+
+        ...
+
+        |-- ApplicationManifest.xml
+
```
Comment (gavinbarron) [inline docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md]: I've re-worked this section and removed mention of this file as it's not needed for this application to work based on my testing

File: docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+```powershell
+
+where node.exe
+
+xcopy "C:\Program Files\nodejs\node.exe" "C:\my\package\NodeService\C"
+
+xcopy "C:\dev\repos\hackathon-starter\*" "C:\my\package\NodeService\C" /s /i
+
+```
+
```
Comment (gavinbarron) [inline docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md]: Done

**Comment by gavinbarron** [comment]
I've re-worked the task where the package for deployment to Service Fabric is built which should eliminate any confusion. Hopefully now you will be able to get this working as intended.

**Comment by OguzPastirmaci** [comment]
Thank you for the changes. Now everything looks good. There wasn't any reference to creating a config folder previously, I guess that was causing the issue.

**Pull request merged**